app/view/button/DrawingButton.js

```diff
@@ -60,7 +60,150 @@ Ext.define('CpsiMapview.view.tool.DrawingButton', {
          * The vector layer key of a polygon layer to snap the start and ends
          * of newly drawn lines to, and return the polygon ids
          */
-        polygonLayerKey: null
+        polygonLayerKey: null,
+
+        /**
+         * If the vertices of the snapped edge shall be shown.
+         */
+        showVerticesOfSnappedEdge: true,
+
+        /**
+         * Style of the editing cursor, before first point is set.
+         */
+        drawBeforeEditingPoint: new ol.style.Style({
+            image: new ol.style.Circle({
```

Comment:
Minor: Styles could be refactored into a dedicated class, in order to ensure better readability of the `DrawingButton` class

Reply:
Good idea. I did it with https://github.com/compassinformatics/cpsi-mapview/pull/578/commits/78769d6a4f510b9ffe8b29cbeef59e3795a98f8d